### DISCUSSION TITLE
Fails to generate image when `config_twemoji: true` is set

### DISCUSSION THREAD

**Discussion by sapphi-red** [lowlighter/metrics#1343] [Q&A] [answered]
`connect ETIMEDOUT 151.139.237.124:443` error happens when `config_twemoji: true` is set.
This started happening from yesterday.

I guess the error is happening here.
https://github.com/lowlighter/metrics/blob/a90044266c8c0e803008b8c7b4fae353d35368ed/source/app/metrics/utils.mjs#L633

IIUC this is due to the fact that max cdn is shutdown (https://github.com/twitter/twemoji/issues/556).
https://github.com/twitter/twemoji-parser/blob/f8312f35fdee0e79874bb50213c4544c14fa759b/src/index.js#L27-L28

**Comment by lowlighter** [comment, marked as answer]
Hi 👋 !

Thanks for reporting this issue, it'll be tracked in #1346

**Reply by sapphi-red** [reply]
Thanks!
